guides/common/modules/proc_applying-configurations.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Applying_configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases Satellite restart is required.
```

Comment:
Your PR should not contain product names like satellite. Please use attributes instead as described in CONTRIBUTING.md

Reply:
Missed that. Thank you.